src/Engine/Renderer/OpenGL/OpenGL.hpp

```diff
@@ -12,6 +12,14 @@ using namespace gl45core;
 #include <glbinding/gl45ext/gl.h>
 using namespace gl45ext;
 
+#include <Core/CoreMacros.hpp>

```

Comment:
No need to include this. CORE_USE_DOUBLE is defined in CompileFlags.cmake add pass to the compiler through -DCORE_USE_DOUBLE if needed. removing this include might keep the compile time lower ...